File: android/app/src/main/java/com/phicdy/totoanticipation/model/Game.java
package com.phicdy.totoanticipation.model;

public class Game {
    public final String homeTeam;
    public final String awayTeam;
    public String homeRanking;
    public String awayRanking;
    public Anticipation anticipation = Anticipation.HOME;
    public enum Anticipation {HOME, AWAY, DRAW};

    public Game(String homeTeam, String awayTeam) {
        this.homeTeam = homeTeam;
        this.awayTeam = awayTeam;
    }
}

File: android/app/src/main/java/com/phicdy/totoanticipation/model/Game.kt
package com.phicdy.totoanticipation.model

class Game(val homeTeam: String, val awayTeam: String) {
    val defaultRank: Int = 0
    var homeRanking: Int = defaultRank
    var awayRanking: Int = defaultRank
    var anticipation = Anticipation.HOME

    enum class Anticipation {
        HOME, AWAY, DRAW
    }
}
